### PR TITLE
Highlight PATCH versions can be used to upgrade

### PR DESCRIPTION
**Pull request by hiddeco**

### CODE REVIEW

File: docs/guides/installation.md [hunk 1]
```diff
@@ -512,6 +512,10 @@ flux create helmrelease nginx \
 
 ## Upgrade
```
Comment (hiddeco) [inline docs/guides/installation.md:514]: Open to better title suggestions.

File: docs/guides/installation.md [hunk 2]
```diff
@@ -512,6 +512,10 @@ flux create helmrelease nginx \
...
 
+!!! note "Versions"
```
Comment (hiddeco) [inline docs/guides/installation.md:515]: Open to better title suggestions.

File: docs/guides/installation.md [hunk 3]
```diff
@@ -512,6 +512,10 @@ flux create helmrelease nginx \
...
+Any PATCH version can be used to upgrade to a new MINOR version. It is for
```
Comment (stefanprodan) [inline docs/guides/installation.md:516]: I would encourage the user of the latest patch release, something like:

Please use the latest PATCH version when upgrade to a new MINOR version.

Comment (hiddeco) [reply docs/guides/installation.md]: Changed the wordings a bit.

**Review by stefanprodan — APPROVED** [review]

File: docs/guides/installation.md [hunk 4]
```diff
@@ -512,6 +512,10 @@ flux create helmrelease nginx \
 
 ## Upgrade
 
+!!! note "Patch versions"
+It is safe and advised to use the latest PATCH version when upgrading to a new
```
Comment (stefanprodan) [inline docs/guides/installation.md:516]: I think this needs to be indented to be included in the note.

**Pull request merged**